src/test/run-pass/union/union-nodrop.rs

```diff
@@ -16,10 +15,10 @@ impl Drop for NeedDrop {
 
 // Constant expressios allow `NoDrop` to go out of scope,
 // unlike a value of the interior type implementing `Drop`.
-static X: () = (NoDrop { inner: NeedDrop }, ()).1;
+static X: () = (NoDrop { inner: ManuallyDrop::new(NeedDrop) }, ()).1;
```

Comment:
Can you repeat this for a `const` item as well as for a `const fn`?

Reply:
Done.

Reply:
Thanks!